Doc/c-api/type.rst

```diff
@@ -106,6 +106,12 @@ Type Objects
        GC protocol itself by at least implementing the
        :c:member:`~PyTypeObject.tp_traverse` handle.
 
+.. c:function:: PyObject* PyType_GetName(PyTypeObject *type)
+
+   Return the type's name. The returned object haven't a prefix of module name.
```

Comment:
I think this would be more understandable to people who aren't that familiar with the differences between ``__name__``/``__qualname__``/``tp_name``, but the behavior would still be clear:
```suggestion
   Return the type's name. Equivalent to getting the type's ``__name__`` attribute.
```

Reply:
make sense :)